parser/prism/Translator.h

```diff
@@ -40,22 +40,23 @@ class Translator final {
     Translator &operator=(Translator &&) = delete;      // Move assignment
     Translator &operator=(const Translator &) = delete; // Copy assignment
 public:
-    Translator(const Parser &parser, core::GlobalState &gs, core::FileRef file)
-        : parser(parser), gs(gs), file(file), uniqueCounterStorage(1), uniqueCounter(&this->uniqueCounterStorage) {}
+    Translator(const Parser &parser, core::GlobalState &gs, core::FileRef file,
+               const std::vector<ParseError> &parseErrors)
```

Comment:
Might be worth considering an `absl::Span` instead of a `const vector<...> &`.

Reply:
Hmmm, what's the rationale?

As far as I understand, spans are for wrapping C raw pointers+lengths to give them a C++ container interface. In this particular case, Prism gives us a linked list, which doesn't fit that model. We just traverse it to copy everything into a vector.

https://github.com/sorbet/sorbet/blob/3e478a8d515bd7dd87f40d7093690bea3702a30d/parser/prism/Parser.cc#L43-L57